package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paypal/checkout-components",
-  "version": "5.0.318",
+  "version": "5.0.318-alpha-message-position.0",
```

Comment:
Not sure if we would want to update the package here.

Reply:
good callout, will push the reverting commit